### PR TITLE
feat: dynamic-dag-sidebar-change

### PR DESCRIPTION
**Pull request by pppppjcc**
## Purpose of the pull request

Modify dynamic component sidebar button styles and presentation logic

## Brief change log

Adding styles

## Verify this pull request

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-ui/src/views/projects/workflow/components/dynamic-dag/dag-sidebar.module.scss [hunk 1]
```diff
@@ -15,9 +15,44 @@
...
  */
 
-.task-item {
-  cursor: pointer;
-  height: 20px;
-  width: 100px;
-  border: 1px solid #1890ff;
-}
\ No newline at end of file
+// .task-item {
```
Comment (caishunfeng) [inline dolphinscheduler-ui/src/views/projects/workflow/components/dynamic-dag/dag-sidebar.module.scss]: Please remove if no use.

Comment (pppppjcc) [reply dolphinscheduler-ui/src/views/projects/workflow/components/dynamic-dag/dag-sidebar.module.scss]: done

**Comment by caishunfeng** [comment]
run ci

**Review by devosend — DISMISSED** [review]
LGTM

**Review by zhongjiajie — APPROVED** [review]

**Pull request merged**